Sources/ResilientDecoding/Resilient.swift

```diff
@@ -37,13 +37,17 @@ public struct Resilient<Value: Decodable>: Decodable {
 
   /**
    Transforms the value of a `Resilient` type.
-   If `self` is a resilient array,  care should be taken to ensure that the `value.count` == `transform(value).count` in order to not break the `results` property.
+   If `self` is a resilient array, care should be taken to ensure that the `value.count` == `transform(value).count` in order to not break the `results` property.
    */
   func map<T>(transform: (Value) -> T) -> Resilient<T> {
     Resilient<T>(transform(wrappedValue), outcome: outcome)
   }
   
   #if DEBUG
+  /**
+   `subscript(dynamicMember:)` is defined in files like `ResilientArray+DecodingOutcome`, and is used to provide certain properties only on `@Resilient` properties of certain types. For instance `errors` and `results` are only present on resilient arrays. The reason we need to use `@dynamicMemberLookup` is so that we can add a generic constraint (which we can to `subscript`, but not to properties).
```

Comment:
>  The reason we need to use `@dynamicMemberLookup` is so that we can add a generic constraint (which we can to `subscript`, but not to properties).

Ah!